mocca-client/src/main/java/com/paypal/mocca/client/MoccaFeignEncoder.java

```diff
@@ -53,6 +70,17 @@ public void encode(Object object, Type bodyType, RequestTemplate template) throw
         }
     }
 
+    void validateVariables(List<Variable> vars) {
+        for (Variable var: vars) {
+            // should we skip "raw" variables?
```

Comment:
I think so

Reply:
Let's do that, let's skip raw variables.

@jamesmartinpp as part of this PR, we will have to change end user documentation as well, to document the usage of Bean Validation. When you do that, please also mention Bean Validation is not supported for raw variables.